pipeline.py

```diff
@@ -2,19 +2,34 @@
 Pipeline for text processing implementation
 """
 
+from pathlib import Path
+
+from constants import ASSETS_PATH
+
 
 class EmptyDirectoryError(Exception):
     """
     No data to process
     """
 
 
+class NotDirectoryError(Exception):
+    """
+    Not a directory
+    """
+
+
 class InconsistentDatasetError(Exception):
     """
     Corrupt data: numeration is expected to start from 1 and to be continuous
     """
 
 
+class FileNotFoundError(Exception):
```

Comment:
it should be a built-in error

Reply:
Deleted this one after PyCharm suggestion.